contrib/blueflood-docker/docker-entrypoint.sh

```diff
@@ -70,13 +70,19 @@ if [ "$INIT_ELASTICSEARCH" != false ]; then
   /ES-Setup/init-es.sh -u $ELASTICSEARCH_HOST:9200 -r false
 fi
 
-printenv > blueflood.conf
+if [ -z "$BLUEFLOOD_CONF_LOCATION" ]; then
+  BLUEFLOOD_CONF_LOCATION="./blueflood.conf"
+  printenv > blueflood.conf
+fi
 
-cat > blueflood-log4j.properties << EOL
+if [ -z "$BLUEFLOOD_LOG4J_CONF_LOCATION" ]; then
+  BLUEFLOOD_LOG4J_CONF_LOCATION="./blueflood-log4j.properties"
+  : ${BLUEFLOOD_LOG_LEVEL=INFO}
```

Comment:
👍  learnt something new. colon syntax

Reply:
That's something I learned about recently too :D